agent/build.gradle.kts

```diff
@@ -179,3 +161,12 @@ tasks {
 rootProject.tasks.named("release") {
   finalizedBy(tasks["publishToSonatype"])
 }
+
+fun CopySpec.isolateClasses(jars: Iterable<File>) {
+  jars.forEach {
+    from(zipTree(it)) {
+      into("inst")
+      rename("(^.*)\\.class\$", "\$1.classdata")
```

Comment:
In my brain I wouldn't consider the starting ^ part of the match group, but if this works it's fine with me. Not a big deal.

Reply:
Wow, I copy-pasted this fragment of code tens of times and didn't even notice that 😂